src/bots/deepseek/DeepSeekAPIChatBot.js

```
@@ -0,0 +1,10 @@
+import DeepSeekAPIBot from "./DeepSeekAPIBot";
+
+export default class DeepSeekAPIChatBot extends DeepSeekAPIBot {
+  static _className = "DeepSeekAPIChatBot";
+  static _model = "deepseek-chat";
+
+  constructor() {
+    super();
+  }
```

Biome

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)